lib/views/components/async/loading_icon_button.dart

```diff
@@ -31,10 +31,9 @@ class LoadingIconButton extends HookWidget {
 
     final stillIconButton = IconButton(
       icon: Icon(icon, size: iconSize),
-      onPressed: () async {
+      onPressed: () {
         isLoading.value = LoadingState.pending;
-        await onClick();
-
+        onClick();
```

Comment:
This is not the same behavior as before. You can't just remove async and awaits. It won't produce the same output.

I advise you to read the following: https://dart.dev/codelabs/async-await

Otherwise, I discussed with @JoachimFavre and we might consider dropping the following DCM rule as it is not very constructive https://dcm.dev/docs/rules/common/avoid-passing-async-when-sync-expected/.

Reply:
See https://github.com/ProximaEPFL/proxima/pull/262#issuecomment-2106350762

Reply:
I reverted the code back to how it was before with [4905711](https://github.com/ProximaEPFL/proxima/pull/262/commits/4905711405317b852f2c4d46faea227b410c3bab)